### PR TITLE
Project-wide Styling

### PR DESCRIPTION
**Pull request by kmeyerhofer**
# Project Styling

## Features
* Blue color scheme with white tree `svg` background for a winter-y feel.
  * Color variables used, simplifying changes if desired.
* Navbar visible on every page, allowing site wide navigation.
* Lots of use of `display: grid;` and `display: flex;` for modularity and ease of positioning.
* Data organizational changes to all `/show.html.erb` pages. (Not counting the numerous `<div>` container additions)

### Branch Requirements
This branch is dependent on the `move-new-item` branch. ( PR #22 )

### CODE REVIEW

**Comment by kmeyerhofer** [comment]
Adds two fonts to the project: `Germania One` and `Glacial Indifference`.

**Comment by meyerhoferc** [comment]
Please resolve merge conflicts here as well

**Comment by kmeyerhofer** [comment]
Merge conflicts fixed. @meyerhoferc

**Comment by meyerhoferc** [comment]
@kmeyerhofer some merge conflicts and tests are not passing. I'm going to make some commits in the interest of time

File: app/controllers/items_controller.rb [hunk 1]
```diff
@@ -10,11 +10,10 @@ def create
     @item.list_id = @list.id
     if authorized_user(@user) && @item.save
       flash[:notice] = "Item created successfully."
-      redirect_to group_list_path(@group, @list)
     else
       flash[:warning] = full_sentence_errors(@item)
-      render 'new'
     end
+    redirect_to group_list_path(@group, @list)
```
Comment (meyerhoferc) [inline app/controllers/items_controller.rb]: @kmeyerhofer Change I made -- was getting a view not found error for `new` form for items. I assumed the desired behavior is to redirect back tot he path and display the appropriate error message

**Pull request merged**